### DISCUSSION TITLE
Claim comission?

### DISCUSSION THREAD

**Discussion by Gribnik2K** [subquery/subql#909] [Frontier TestNet - Bug Reporting]
All rewards are "claimed"
But there is no opportunity to interact (stay, unstake, commission change)....
![image](https://user-images.githubusercontent.com/42679698/164946929-7eaf1544-e350-4d4b-82ce-2bb6dac3e4ef.png)
![image](https://user-images.githubusercontent.com/42679698/164946941-0dcf502c-5cdf-46ab-bfeb-a34558df5945.png)
![image](https://user-images.githubusercontent.com/42679698/164947004-2d7dedaf-7d14-467b-8b70-6d77ed7d2155.png)

**Comment by ianhe8x** [comment]
As the error msg suggested, it's due to indexer failed to call some maintanence tx. It is possible for delegator to exit even in this situation, only we haven't added it into explorer. It is planned in future release.
